_data/sidebar.yml

```diff
@@ -35,34 +43,42 @@
   isSectionHeader: 0
   sectionTitle:
   subgroup: 0
-
+  
 - sbSecId: 0
-  title: Contributing
-  link: /contributing/contribute.html
+  title: Members and Partners
+  link: /partners/partners.html
   isHeader: 0
   isSectionHeader: 0
   sectionTitle:
   subgroup: 0
-
+  
 - sbSecId: 0
-  title: About Prebid.org
-  link: /overview/what-is-prebid-org.html
+  title: Prebid Management Committees
+  link: /overview/prebid-management-committees.html
   isHeader: 0
   isSectionHeader: 0
   sectionTitle:
   subgroup: 0
-
+  
 - sbSecId: 0
-  title: Members and Partners
-  link: /partners/partners.html
+  title: Developers
+  link: /developers.html
   isHeader: 0
   isSectionHeader: 0
-  sectionTitle:
   subgroup: 0
-
+  
 - sbSecId: 0
-  title: Prebid Management Committees
-  link: /overview/prebid-management-committees.html
+  title: "<span style='color:#EA9622;'>Policies</span>"
```

Comment:
The links under here should be indented like they are under the video examples.

Reply:
Fixed.